src/main/java/com/minecolonies/coremod/client/render/worldevent/RenderTypes.java

```diff
@@ -64,7 +61,7 @@ private InnerRenderTypes(final String nameIn,
                     .setShaderState(POSITION_TEX_SHADER)
                     .setTextureState(new RenderStateShard.TextureStateShard(p_173202_, false, false))
                     .setTransparencyState(TRANSLUCENT_TRANSPARENCY)
-                    .setDepthTestState(ALWAYS_DEPTH_TEST)
```

Comment:
not this one xd, cuz no_depth_test provided by vanilla has shadow bug cuz of driver optimizations

Reply:
Maybe they ran into the same bug and changed it at some point.  Because I did check, and `NO_DEPTH_TEST` is the same as passing `enableDepthTest()` with `GL_ALWAYS`.

Reply:
Or apparently not.  It looked like it, but turns out they skip that for `GL_ALWAYS` after all.  Reverted it.